src/components/shared/TreeView.js

```diff
@@ -156,23 +156,25 @@ class TreeViewRowScrolledColumns extends React.PureComponent<
     (this: any)._onClick = this._onClick.bind(this);
   }
 
-  _onClick(event: SyntheticMouseEvent<Element>) {
+  /**
+   * In this mousedown handler, we use event delegation so we have to use
+   * `target` instead of `currentTarget`.
+   */
+  _onClick(event: { target: Element } & SyntheticMouseEvent<Element>) {
```

Comment:
same thing here, I reintroduced the use of `target` because of event delegation.

Reply:
Note that with newer Flow, the target needs to be __before__ the `SyntheticEvent`.

Reply:
Weird, why is that?

Reply:
in SyntheticEvent, there is also a `target` property. Looks like Flow doesn't know how to merge `{ prop: Type1 } | { prop: Type2 }` into `{ prop: Type1 | Type 2 }` and instead just use the first one. So that's why :)

I think there is an issue at Flow about that but I can't find it ATM.